cdap-proto/src/main/java/co/cask/cdap/proto/Id.java

```diff
@@ -427,7 +427,8 @@ private Stream(final Namespace namespace, final String streamName) {
       Preconditions.checkNotNull(streamName, "Stream name cannot be null.");
 
       Preconditions.checkArgument(isId(streamName),
-                                  "Stream name can only contains alphanumeric, '-' and '_' characters only.");
+                                  String.format("Stream name can only contains alphanumeric, " +
```

Comment:
Not introduced in your PR, but s/contains/contain/, also there is an extra 'only' in the end.